tests/foreman/ui/test_reporttemplates.py

```diff
@@ -210,7 +210,18 @@ def test_positive_end_to_end(session, module_org, module_loc):
         session.reporttemplate.update(name, {'template.name': new_name, 'type.snippet': True})
         rt = session.reporttemplate.read(new_name)
         assert rt['template']['name'] == new_name
+        assert rt['template']['default'] is False
+        assert rt['template']['template_editor']['editor'] == content
+        assert rt['inputs'][0]['name'] == template_input[0]['name']
```

Comment:
I did not understand why template_input was kept in list with single object. may be adding as dict would have been simpler ? non-blocking comment and not regarding this PR though. But Just a Thought.

Reply:
Because Airgun uses `RemovableWidgetsItemsListView` there. Haven't looked at code, but judging by name it expects list of items to fill. So you would have to add more complexity to Airgun, so it can handle dict passed in directly. Or you can just wrap your dict in list on test side.